### ISSUE TITLE
[Android][Accessibility] Accessible video controls

### ISSUE THREAD

**Issue by golddove** [microsoft/AdaptiveCards#5150]
# Platform

- [ ] Android

# Version of SDK

latest main

# Details

Source issue: https://microsoft.visualstudio.com/OS/_workitems/edit/28383710

Video player controls (play/pause) should be accessible with screen reader / keyboard navigation on Android.
